src/dso_api/router.py

```diff
@@ -0,0 +1,30 @@
+# Route all read requests to the replica databases
+# Route all write requests to the default database
+
+import random
+import environ
+
+env = environ.Env()
+
+replicas = []
+for replica in range(1, 11):
```

Comment:
kunnen we hardcoded 11 in een variabele gieten?

Reply:
Yes, ik heb de MAX_REPLICA_COUNT toegevoegd zodat het aantal replicas kan worden aangepast zonder de DSO-API code aan te passen.